modules/fb_instant_articles_display/src/EntityPropertyMapper.php

```diff
@@ -109,16 +109,27 @@ private function addHeaderFromProperties() {
       );
     }
 
+    // user token or static string for author name
+    $author = '';
+    if (module_exists('Token')) {
+      $display_author = variable_get('fb_instant_articles_display_author', '[node:author]');
+      $author = token_replace($author_token, array('node' => $node));
+    } else {
```

Comment:
Move the `else` control to a new line. See Drupal's [Coding standards / Control Structures](https://www.drupal.org/coding-standards#controlstruct).